FieldOpt/Optimization/objective/NPV.cpp

```diff
@@ -0,0 +1,123 @@
+//
```

Comment:
Add the GNU GPLv3 license header comment in new files. You can just copy it from another source file and change the name.